docs/blog/2019-02-26-getting-started-with-gatsby-themes/index.md

```diff
@@ -36,58 +36,43 @@ Tidy up your _package.json_ file and create workspaces which includes the projec
 
 Next, you want to create your _site_ directory and your _packages_ directory within your _gatsby-theme_ project directory. Make sure the names that you choose for your directories are the same as what you put in your workspaces. You will also want to go into your packages directory and make another directory with the name of your theme. For the purpose of this tutorial, we will call it _theme_. Then you will want to `yarn init` the _theme_ directory and the _site_ directory.
 
-```
+```sh
 mkdir site
 mkdir packages
 cd packages
 mkdir theme
 cd theme
-yarn init
+yarn init -y
+touch index.js
 cd ../../site/
-yarn init
+yarn init -y
 ```
 
 After using `yarn init` you will be asked a few questions. The main thing you will want to pay attention to is the entry point (index.js). For the _theme_ directory, you can leave the entry point as _index.js_ (just make sure you have an _index.js_ file) and for the _site_ directory, you can make entry point _gatsby-config.js_. Next, you will want to go into your _site_ directory and do `yarn workspace site add gatsby`.
```

Comment:
this one isn't really correct anymore. as -y doesn't ask questions. maybe rewrite to if you use yarn init without -y it will ask a few questions?

Reply:
Ah good point, will remove the `-y` in this section.

Reply:
Actually, going to clarify about the `-y` as you've suggested so that way one can cut and paste the whole block of commands and run in one go : )